src/simularium/VisData.ts

```diff
@@ -198,7 +198,7 @@ class VisData {
         let j = AGENTS_OFFSET;
         for (let i = 0; i < expectedNumAgents; i++) {
             const agentData: AgentData = {
-                visType: -1,
+                "vis-type": -1,
```

Comment:
Whats the reason for this change? Was this not working correctly before?

Reply:
unfortunately, the server provides this data in json as "vis-type".  I'm just making it all consistent here in keeping that name everyplace.

Reply:
I suppose the alternative is to go the other way and modify it as soon as it's read, which would probably be better but a bit uglier at parse time.

Reply:
Ok, and all the other names here are how the server provides them? I was just surprised this is `vis-type` while there are other camelCase names

Reply:
it has been like that since early days.  I have a feeling Eric named it from the server side before the js code was even written.